src/audio/eq_fir/eq_fir.c

```diff
@@ -61,11 +61,13 @@
 /* src component private data */
 struct comp_data {
 	struct fir_state_32x16 fir[PLATFORM_MAX_CHANNELS]; /**< filters state */
-	struct sof_eq_fir_config *config; /**< pointer to setup blob */
-	enum sof_ipc_frame source_format; /**< source frame format */
-	enum sof_ipc_frame sink_format;   /**< sink frame format */
-	int32_t *fir_delay;		  /**< pointer to allocated RAM */
-	size_t fir_delay_size;		  /**< allocated size */
+	struct sof_eq_fir_config *config;	/**< pointer to setup blob */
```

Comment:
Whitespace change?

Reply:
Yes I thought the block of code looks better with similar indent, OK?

Reply:
Yeah just figured it'd be nice if it's separate so it's easy to see that you just add one member rather than doing anything nontrivial. Not something to fuss over though.

Reply:
I'll do anyway one more round due to other finding here and this would be low effort. If you think the white space adjust commit is feasible please let me know!

Reply:
OK to keep as is?

Reply:
I can accept it since this is the only place where such changes are combined. It does make the diff a bit confusing but it can be understood.

Reply:
Thanks! I'll resolve this.